Makefile

```diff
@@ -242,6 +242,7 @@ protoc: $(PROTOGEN) $(MOCKGEN) $(GOIMPORTS) $(PROTOC_GEN_GO) $(PROTOC_GEN_GO_GRP
 		PROTOGEN=$(PROTOGEN) MOCKGEN=$(MOCKGEN) GOIMPORTS=$(GOIMPORTS) \
 		API_BINPB=$(API_BINPB) PROTO_ROOT=$(PROTO_ROOT) PROTO_OUT=$(PROTO_OUT) \
 		./develop/protoc.sh
+	@go generate -run gensearchattributehelpers ./common/searchattribute/...
```

Comment:
@dnr, I don't think this is a right place for this line. Should we combine all codegen (besides mocks) under the single make target?

Reply:
I moved it.